samples/opencl/FindOpenCL.cmake

```diff
@@ -0,0 +1,174 @@
+# Find OpenCL
```

Comment:
I have no significant issues with CMake 3.0.2 on Linux.
Also in another comment I suggested to use **standalone** build for this sample, because OpenCV uses self-contained specific OpenCL finder and file like this can break OpenCV build.